### PR TITLE
Make lookup of statically built binary more robust

### PR DESCRIPTION
**Pull request by locker**
Needed for [Tarantool EE](https://github.com/tarantool/tarantool-ee).

### CODE REVIEW

**Review by Totktonada — APPROVED** [review]
I don't mind, but it would be good to know a reason. What works bad with the old code?

**Comment by locker** [comment]
> I don't mind, but it would be good to know a reason. What works bad with the old code?

I mentioned ctest and EE in the commit message to clear up the purpose of this change.

File: static-build/CMakeLists.txt [hunk 1]
```diff
@@ -303,31 +303,31 @@ ExternalProject_Add(tarantool
 )
 
 enable_testing()
-ExternalProject_Get_Property(tarantool binary_dir)
-SET(TARANTOOL_BINARY_DIR ${binary_dir})
+ExternalProject_Get_Property(tarantool install_dir)
```
Comment (sergos) [inline static-build/CMakeLists.txt:306]: does it mean one have to run `make install` to have an ability to run `make test`?

Comment (locker) [reply static-build/CMakeLists.txt:306]: static-build tests are run with the `ctest` command, not `make test`, see https://github.com/tarantool/tarantool/blob/master/static-build/README.md#usage

`ctest` requires `cmake` and `make` to be run.

**Pull request merged**